sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -50,7 +51,7 @@ class BlobAPITest extends APISpec {
         HttpHeaders headers = response.headers()
 
         then:
-        body == defaultData
+        defaultData.compareTo(body) == 0
```

Comment:
This was already idiomatic to groovy. `==` checks to see if `Comparable` is implemented and calls `objA.compareTo(objB) == 0` if so. Otherwise, it calls `objA.equals(objB)`.

(future reference: if you want to compare reference equality in groovy, you write `objA.is(objB)`)

Reply:
Revert the changes back. Thanks for the info!